### DISCUSSION TITLE
Can Connector-X query from a SAP HANA database?

### DISCUSSION THREAD

**Discussion by nvidia-wtruong** [sfu-db/connector-x#648] [Q&A]
Hi, I'm dependent on SAP HANA for my data sources. Does connector-x support HANA?

**Comment by wangxiaoying** [comment]
Hi @nvidia-wtruong , we don't support SAP HANA right now. To do it, one needs to [add a hana source](https://github.com/sfu-db/connector-x/blob/main/CONTRIBUTING.md#how-to-add-a-new-source) with a corresponding driver in rust like [hdbconnect](https://docs.rs/hdbconnect/latest/hdbconnect/).

**Comment by avhz** [comment]
I would also benefit from SAP HANA support.
